### ISSUE TITLE
Playnite 9 support

### ISSUE THREAD

**Issue by ld892012** [Deytron/PlayOS#1]
It's possible to add Playnite 9 support for this beautiful theme ?

**Comment by BluePhoenix** [comment]
Agreed, definitely the one thing I miss in fullscreen after upgrading to Playnite 9.
